server/public/main.css

```diff
@@ -79,11 +84,11 @@ input {
   margin-left: 0;
 }
 
-.navbar-menu button {
+#login-button {
```

Comment:
Do we need to use an id here or would a class suffice?

Reply:
Removed id